GTG/gtk/editor/editor.py

```diff
@@ -536,14 +614,28 @@ def inserttag_clicked(self, widget):
             self.textview.insert_text("@", itera)
         else:
             self.textview.insert_text(" @", itera)
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
 
     def inserttag(self, widget, tag):
         self.textview.insert_tags([tag])
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
```

Comment:
No, it is used only once (I asked you remove that block of code from refresh_editor that calls it, as you don't need it any more)